src/main/resources/checkstyle-config.xml

```diff
@@ -260,6 +260,7 @@
     <module name="AbbreviationAsWordInName">
       <property name="ignoreFinal" value="false"/>
       <property name="allowedAbbreviationLength" value="0"/>
+      <property name="allowedAbbreviations" value="DTO,ETL" />
```

Comment:
Ooooh. Didn't realise you could do this.
Might be worth having a look through the checkstyle report and see if there are any other common abbreviations that we might want to allow?
URN and DO might be good ones to consider? There's a few other instances of things like HTML, CSV, URL, 2FA/MFA and so on that I'm less sure about...
Is there a way we could do something about all the abstract classes starting with I as well? Does the values field support regex or is in plain text only?